packages/sdk/src/search/usePagination.ts

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from 'react'
+
+import { format } from './serializer'
+import { useSearch } from './useSearch'
+import type { State as SearchState } from './useSearchState'
+
+const getLink = (state: SearchState) => {
+  const { pathname, search } = format(state)
+
+  return `${pathname}${search}`
+}
+
+export const usePagination = (totalItems: number) => {
+  const { pages, itemsPerPage, state } = useSearch()
+
+  const total = Math.ceil(totalItems / itemsPerPage)
+  const next = Number(pages[pages.length - 1]) + 1
+  const prev = pages[0] - 1
```

Comment:
Not really, The previous page is `0` because you will be rendering pages `1`, `2` and `3`. For the user who's seeing the page, the previous page is the page `0`